frontend/src/shared/components/select-users/Models.ts

```diff
@@ -1,33 +1,33 @@
 export enum BusinessRole {
```

Comment:
Not sure that this entity will be enum. It can be as table in db too. Ask our coaches or QA's about it.